draft-ar-pquip-pqc-engineers.md

```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
 The fourth-round of the NIST process focuses only on KEMs. The goal of that round is to select an althernative algorithm that is based on different hard problem than Kyber.
 The candidates still advancing for standardization are:
 
-* [Classic McEliece](https://classic.mceliece.org/)
-* [BIKE](https://bikesuite.org/)
-* [HQC](http://pqc-hqc.org/)
+* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes
```

Comment:
I suggest a line about Goppa code. For instance, we can say "Goppa codes are a class of error-correcting codes that can correct a certain number of errors in a transmitted message. The decoding problem involves recovering the original message from the received noisy codeword."